README.md

```diff
@@ -89,9 +89,6 @@ To run this sample, you'll need:
 
 ## Steps to Run the app
 
-> [!NOTE]
-> This sample ships with a default `redirect_uri` configured in the `AndroidManifest.xml`. In order for the default `redirect_uri` to work, this project must be built with the `debug.keystore` located in the `gradle/` directory. To configure signing in Android Studio, see [Sign Your App](https://developer.android.com/studio/publish/app-signing).
-

```

Comment:
The part about having a default redirect_uri configured might be helpful. This might be a bit wordy. What do you think of:

As a convenience, this sample ships with a default `redirect_uri` preconfigured in the `AndroidManifest.xml` so that you don't have to first register your own app object. A `redirect_uri` is partly based on your app's signing key. The sample project is configured to sign using the included `debug.keystore` so that the provided `redirect_uri` will work.

Reply:
Oh, and the casing is wrong. It should be "Steps to run the app"

Reply:
Sounds good. will do that.